src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpInteractiveBoxSelection.cs

```diff
@@ -24,12 +24,17 @@ public CSharpInteractiveBoxSelection(VisualStudioInstanceFactory instanceFactory
 
         protected override void Dispose(bool disposing)
         {
-            if (disposing)
+            try
             {
-                VisualStudio.ExecuteCommand(WellKnownCommandNames.Edit_SelectionCancel);
+                if (disposing && VisualStudio != null)
+                {
+                    VisualStudio.ExecuteCommand(WellKnownCommandNames.Edit_SelectionCancel);
```

Comment:
Why not `VisualStudio?.ExecuteCommand` ?

Reply:
I had that to start with but ended up opting for consistency with the other derived implementation in `CSharpReplIdeFeatures`.